app/components/avo/base_component.rb

```diff
@@ -1,46 +1,66 @@
 # frozen_string_literal: true
 
-class Avo::BaseComponent < ViewComponent::Base
-  include Turbo::FramesHelper
+module Avo
```

Comment:
Hey, it seems that a few styling changes ended up in this PR.
Could I ask you to revert them as they were? It makes evaluating this PR much more difficult.